docs/hugo/content/guide/annotations.md

```diff
@@ -21,6 +21,16 @@ it is _not_ deleted in Azure. In REST API terminology, PUT and GET are allowed w
     
 Unknown values default to `manage`.
 
+### `serviceoperator.azure.com/credential-from`
+
+Instructs the operator to use a particular credential for the resource. This credential supersedes any global or
+namespace scoped credentials the operator has configured.
```

Comment:
Currently, this reads as though the secret should be in the annotation.

```suggestion
Instructs the operator to read the credential for the resource from the specified secret. This credential supersedes any global or namespace scoped credentials the operator has configured.
```

Reply:
Done